Examples/Algorithms/Vertexing/src/VertexFitAlgorithm.cpp

```diff
@@ -61,6 +61,16 @@ FW::ProcessCode FWE::VertexFitAlgorithm::execute(
   for (auto& vertexAndTracks : input) {
     const auto& inputTrackCollection = vertexAndTracks.tracks;
 
+    // Only fit vertices where true vertex is indeed close to beam line
+    // This removed secondaries and odd vertices that are not wanted in
+    // this example
+    if (std::sqrt(vertexAndTracks.vertex.position().x() *
+                      vertexAndTracks.vertex.position().x() +
+                  vertexAndTracks.vertex.position().y() *
+                      vertexAndTracks.vertex.position().y()) > 4_mm) {
```

Comment:
hard coded limits?
you could also check for 16 and not do the sqrt

Reply:
Updated to use a configurable limit. It's easier to read using the sqrt I would say and since it's only an example algorithm in the Examples, performance is not an issue I think